### PR TITLE
New package: mbusd

### PR DESCRIPTION
**Pull request by marcin1j**
Maintainer: me
Compile tested: (rampis, Nexx WT3020 (MT7620), OpenWrt master (r14531))
Run tested: (rampis, Nexx WT3020 (MT7620), OpenWrt master (r14531))

Description:
Mbusd is open-source Modbus TCP to Modbus RTU (RS-232/485) gateway.

### CODE REVIEW

File: net/mbusd/Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+PKG_NAME:=mbusd
+PKG_VERSION:=0.4.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/3cky/mbusd/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=8458afc455a28c8f798cadd3982f9e03496a494a2269e31e8255a6ca273a6898
+
+PKG_LICENSE:=BSD-3c
```
Comment (neheb) [inline net/mbusd/Makefile:18]: BSD-3-Clause , as per SPDX.

File: net/mbusd/Makefile [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2014 OpenWrt.org
```
Comment (karlp) [inline net/mbusd/Makefile:2]: This should either be you, or omitted, unless you have copyright assignment agreements with openwrt, and unless they were done in 2014.

**Comment by marcin1j** [comment]
@neheb @karlp 

Fixed. Sorry for the delay.

File: net/mbusd/files/mbusd.init [hunk 3]
```diff
@@ -0,0 +1,68 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2019 OpenWrt.org
```
Comment (karlp) [inline net/mbusd/files/mbusd.init:2]: this was also an invalid copyright assignment

Comment (marcin1j) [reply net/mbusd/files/mbusd.init:2]: Removed

**Pull request merged**